charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -86,6 +86,10 @@ data:
         taskUsageMetricsEnabled: {{ .Values.global.taskUsageMetricsEnabled }}
         taskUsageReportNumberOfDays: {{ .Values.houston.cleanupTaskUsageData.olderThan}}
 
+      deployRollback:
+        enabled: {{ .Values.global.deployRollbackEnabled }}
+        deployRevisionReportNumberOfDays: {{ .Values.houston.cleanupDeployRevisions.olderThan}}
+
```

Comment:
enableUpdateDeploymentImageEndpoint this flag also to be included when rollback feature  is enabled